airgun/views/host.py

```diff
@@ -284,16 +350,76 @@ def is_displayed(self):
             self.title, exception=False) is not None
 
 
-class HostsChangeEnvironment(BaseLoggedInView):
+class HostsChangeGroup(HostsActionCommonDialog):
+    title = Text(
+        "//h4[text()='Change Group - The"
+        " following hosts are about to be changed']")
+    host_group = Select(id='hostgroup_id')
+
+
+class HostsChangeEnvironment(HostsActionCommonDialog):
     title = Text(
         "//h4[text()='Change Environment - The"
         " following hosts are about to be changed']")
-    table = SatTable("//div[@class='modal-body']//table")
-    keep_selected = Checkbox(id='keep_selected')
     environment = Select(id='environment_id')
-    submit = Text('//button[@onclick="submit_modal_form()"]')
+
+
+class HostsTaxonomyMismatchRadioGroup(View):
+    """Handle Taxonomy Mismatch Radio Group
+
+        Example html representation::
+
+            <form ...>
+                <div class="clearfix">
+                    ...
+                </div>
+                <input type="radio" id="location_optimistic_import_yes" ..>
+                 Fix Location on Mismatch
+                <input type="radio" id="location_optimistic_import_no" ..>
+                 Fail on Mismatch
+            </form>
+        """
+    fix_mismatch = Text("//input[contains(@id, 'optimistic_import_yes')]")
```

Comment:
@id=optimistic_import_yes

Reply:
The full id for assign location looks like: location_optimistic_import_yes
for assign organization: organization_optimistic_import_yes